api/controllers/routes.go

```diff
@@ -20,10 +20,12 @@ func NewRouter() (router *mux.Router) {
 	router.HandleFunc("/apps/{app}", api("app.delete", AppDelete)).Methods("DELETE")
 	router.HandleFunc("/apps/{app}/builds", api("build.list", BuildList)).Methods("GET")
 	router.HandleFunc("/apps/{app}/builds", api("build.create", BuildCreate)).Methods("POST")
+	router.HandleFunc("/apps/{app}/builds/import", api("build.import", BuildImport)).Methods("POST")
```

Comment:
Definitely could. I was thinking of having Import return a release so that the newly imported build could be released. Thoughts on that since a code path could return a release struct vs the others returning a build. Seems a little off since the function is called `BuildCreate`.